src/content/posts/new-wireguard-mode/index.md

```diff
@@ -0,0 +1,97 @@
+---
+title: "A more user-friendly transparent mode, based on WireGuard"
+summary: | 
+    The next release of mitmproxy is slated to ship with a new alternative to the existing "transparent" mode, based on
+    WireGuard. The new "wireguard" mode was designed to be much simpler to set up and use, and in most cases, it does not
+    require taking any actions with elevated privileges.
+date: 2022-09-23
+weight: 10
+tags: [tech]
+author:
+    name: Fabio Valentini
+    twitter: decathorpe
+---
+
+The next release of mitmproxy is slated to ship with a new alternative to the existing `transparent` mode, based on
+WireGuard. The new `wireguard` mode was designed to be much simpler to set up and use, and in most cases, it does not
+require taking any actions with elevated privileges.
+
+Connecting another device and routing its traffic through mitmproxy becomes as simple as connecting to a WireGuard VPN
+--- either by using the WireGuard configuration that is printed by mitmproxy, or by scanning the QR code that is shown
+in the mitmweb interface.
+
+## How it works
+
+The core of this feature is implemented in the [`mitmproxy-wireguard`](https://pypi.org/project/mitmproxy-wireguard/)
```

Comment:
will the main mitmproxy package pull in this dependency automatically (for all users), or do interested users need to install mitmproxy-wireguard explicitly?

Reply:
It's a new dependency and will be included by default. I added a sentence to clarify this.